docs/20-web/10-getting-started.md

```diff
@@ -0,0 +1,90 @@
+---
+title: 'Getting Started'
+sidebar_position: 1
+---
+
+
+## Installation
+
+### Docker
+
+Make sure to have Docker, Docker-Compose, Node.js and pnpm installed.
```

Comment:
Should we add a short introduction stating that this is only necessary for development? One might get the impression that the web client always needs to be installed.